### PR TITLE
Answer open question 1

### PR DESCRIPTION
**Pull request by CBinh710**
Answer open question 1: "Should we use tables in HTML? Why?"

### CODE REVIEW

File: OpenQuestion/answer-question-1.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (hungdo-agilityio) [inline OpenQuestion/answer-question-1.md]: Remove empty line here

File: OpenQuestion/answer-question-1.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# Should we use tables in HTML? Why?
+
+## We should use tables in HTML incase
+Tabular Data Presentation: Use tables when you need to show data in a structured grid format with rows and columns.
+Comparison and Cross-Referencing: For instance, if you’re comparing product features, specifications, or prices, a table layout works well.
+Legacy Layouts and Compatibility:In older web development practices, tables were commonly used for page layout.
```
Comment (hungdo-agilityio) [inline OpenQuestion/answer-question-1.md]: Use unorder list for more readable
https://www.markdownguide.org/basic-syntax/#unordered-lists

Comment (hungdo-agilityio) [inline OpenQuestion/answer-question-1.md]: `Compatibility:In` -> `Compatibility: In` Please add space after `:`

File: OpenQuestion/answer-question-1.md (hunk 1 above)
Comment (CBinh710) [reply OpenQuestion/answer-question-1.md]: thanks, i fixed it

File: OpenQuestion/answer-question-1.md (hunk 2 above)
Comment (CBinh710) [reply OpenQuestion/answer-question-1.md]: thanks, i added it

Comment (CBinh710) [reply OpenQuestion/answer-question-1.md]: thanks, i fixed it

Comment (hungdo-agilityio) [reply OpenQuestion/answer-question-1.md]: @CBinh710 Still not resolved yet

**Pull request merged**